datasets/geoschem-input-data.yaml

```diff
@@ -0,0 +1,42 @@
+Name: GEOS-Chem Input Data
+Description: Input data for the GEOS-Chem Chemical Transport Model, includes NASA/GMAO MERRA-2 and GEOS-FP [meteorological products](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#met), [chemistry input data](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#chemistry-input-data), [emissions input data](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#emis-inputs), and other smaller datasets such as model [initial conditions](https://geos-chem.readthedocs.io/en/latest/gcclassic-user-guide/input-overview.html#initial-conditions-input-data>).
+Documentation: https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/geos-chem-input-data-on-aws.html
+Contact: https://geoschem.github.io/support-team.html
+UpdateFrequency: New meteorological and emission data will be added when available.
+ManagedBy: "[Washington University in St. Louis](https://wustl.edu) and [Harvard University](https://www.harvard.edu/) and"
+Collabs:
+  ASDI:
+    Tags:
+      - air quality
+Tags:
+  - aws-pds
+  - climate
+  - weather
+  - meteorological
+  - environmental
+  - air quality
+  - chemistry
+  - atmosphere
+  - model
+License: https://geoschem.github.io/license.html
+Resources:
+  - Description: Top-level directory for all GEOS-Chem data.
+    ARN: unknown
+    Region: us-east-1
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+    - Title: Getting started with GEOS-Chem Input Data
+      URL: https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/getting-started-aws.html
+      AuthorName: GEOS-Chem Support Team
+      AuthorURL: http://geos-chem.org/support-team
+    - Title: GEOS-Chem Classic Quickstart Guide
+      URL: https://geos-chem.readthedocs.io/en/latest/getting-started/quick-start.html
+      AuthorName: GEOS-Chem Support Team
+      AuthorURL: http://geos-chem.org/support-team
+    - Title: GCHP Quickstart Guide
+      URL: https://gchp.readthedocs.io/en/latest/getting-started/quick-start.html
+      AuthorName: GEOS-Chem Support Team
+      AuthorURL: http://geos-chem.org/support-team
+  Tools & Applications:
+  Publications:
```

Comment:
Could you add links to the GCClassic and GCHP on the cloud papers? I think it's OK that these technically used the older S3 bucket.

```
    - Title: "Enabling Immediate Access to Earth Science Models through Cloud Computing: Application to the GEOS-Chem Model"
      URL: https://doi.org/10.1175/BAMS-D-18-0243.1
      AuthorName: Jiawei Zhuang et al.
    - Title: "Enabling high-performance cloud computing for Earth science modeling on over a thousand cores: application to the GEOS-Chem atmospheric chemistry model"
      URL: https://doi.org/10.1029/2020MS002064
      AuthorName: Jiawei Zhuang et al.
```